pallets/nft/src/lib.rs

```diff
@@ -399,6 +399,8 @@ impl<T: Config> Module<T> {
             NumberOfNFTs::get(nfts.ticker(), sender_portfolio.did) >= transferred_amount,
             Error::<T>::InvalidNFTTransferNoBalance
```

Comment:
Minor, but "Balance" and "amount" doesn't make sense for NFT.  "Count" would be better.